content/content/pages/vaccine-info.md

```diff
@@ -1,17 +1,23 @@
 Title: Where can I get the COVID vaccine in Chicago?
-Date: 03-19-2021
-Tags: vaccine
+Date: 04-03-2021
+Tags: vaccine, covid
 
-On March 29, Chicago will enter Covid vaccine [phase 1C](https://www.chicago.gov/city/en/sites/covid19-vaccine/home/vaccine-distribution-phases.html). This phase will include most Chicago residents, such as those working in retail, media, IT, communications, and real estate. There’s a good write-up on the current state of the vaccination effort [here](https://thetriibe.com/2021/03/most-chicagoans-will-be-eligible-for-vaccine-beginning-march-29-but-not-everyone-will-be-able-to-get-vaccinated-immediately/). All that said, many could likely get vaccinated before the mentioned date via one of the following methods:
+On March 29 Chicago entered Covid vaccine [phase 1C](https://www.chicago.gov/city/en/sites/covid19-vaccine/home/vaccine-distribution-phases.html) which made all essential workers and adults with underlying medical conditions eligible for COVID-19 vaccines. On April 12, vaccines will technically be made available to any [Illinois resident](https://coronavirus.illinois.gov/s/). This means that vaccines appointments may be difficult to get during April in Chicago.
```

Comment:
`which made all essential workers`
I've had individuals be confused, not considering themselves essential workers even when they qualify. I think throwing a few examples on here could be beneficial, something like what was there before:
` will include most Chicago residents, such as those working in retail, media, IT, communications, and real estate`

Reply:
That said, it's good to update the date on this!